upstream.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = cpputest
```

Comment:
I believe it should be `cpputest-4.0` based on the directory name in the archive.